bin/virtme-prep-kdir-mods

```diff
@@ -25,6 +25,9 @@ ln -srfT . "$MODDIR/build"
 # to the build kenrnel.
 find "$MODDIR/kernel" -type l -print0 |xargs -0 rm -f --
 while read -r i; do
+    # from v6.2, modules.order lists .o files, we need the .ko ones
+    i=$(echo "$i" | sed 's:\.o$:.ko:')
```

Comment:
Instead of fork/exec-ing a new sed process for every line, this could be done more efficiently by streaming the whole `modules.order` through a single process instead, e.g.:

```sh
sed 's:\.o$:.ko:' modules.order | while read -r i; do
   # existing loop body
done
```

Or alternately, the string substitution could be achieved via parameter expansion instead of sed, e.g.:

```sh
if [ "${i%.o}" != "$i" ]; then
    i="${i%.o}.ko"
fi
```

Reply:
I initially wanted to do the conversion with `${i/%.o/.ko}` but `sh` is used here. I then added the `sed` there to imitate what is done the line below, more to have something simple.

I agree it is not optimal, I didn't think it would have been more important to have it more efficient. I can change of course, thank you for the suggestion.